pkgs/flox-cli/default.nix

```diff
@@ -42,7 +42,7 @@
         then "flox-watchdog"
         else "${flox-watchdog}/bin/flox-watchdog";
 
-      FLOX_ZDOTDIR = flox-activation-scripts + activate.d/zdotdir;
+      FLOX_ZDOTDIR = flox-activation-scripts + "/activate.d/zdotdir";
```

Comment:
(non-blocking) What did this fix? Should we apply the same change to https://github.com/flox/flox/blob/eabea829aa88bd597c87f229e74a6e310db8d1a6/pkgs/rust-pkg/default.nix#L67 if it's even used?